### DISCUSSION TITLE
Gsap timeline problem

### DISCUSSION THREAD

**Discussion by cyprianwaclaw** [vercel/vercel#8437] [Help]
I have a problem with production vercel environment. When I try in dev gsap.timeline is work. Next I send to vercel, i have error 500, gsap.timeline is not a function. How I fix this ?

 

My vercel: [500 - Internal Server Error | Nuxt (cyprianwaclaw.vercel.app)](https://cyprianwaclaw.vercel.app/)

Github project: [cyprianwaclaw/Nuxt-Pinia-ts (github.com)](https://github.com/cyprianwaclaw/Nuxt-Pinia-ts)

**Comment by amyegan** [comment]
Hi @cyprianwaclaw! I don't see a 500 error on your site now. Did you already find a solution?
